downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GSEAOutput.java

```diff
@@ -0,0 +1,92 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+import org.reactome.gsea.ReactomeToMsigDBExport;
+
+public class GSEAOutput {
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void execute(MySQLAdaptor dba, String releaseNumber) throws Exception {
+		logger.info("Running GSEAOutput...");
+		String outFilename = "ReactomePathways.gmt";
+		
+		ReactomeToMsigDBExport exporter = new ReactomeToMsigDBExport();
+		exporter.setIsForGMT(true);
+		exporter.setDBA(dba);
+		exporter.export(outFilename);
+		
+		logger.info("Updating ReactomePathways.gmt with 'Reactome Pathway' column...");
+		// Initial output file needs to be updated so that the third column contains 'Reactome Pathway' 
+		try {
+			FileReader fr = new FileReader(outFilename);
+			BufferedReader br = new BufferedReader(fr);
```

Comment:
The FileReader can be created directly inside the BufferedReader constructor call.

Reply:
If creating a `FileReader` throws an exception, you'll get a stack trace pointing to line 38. If creating a `BufferedReader` throws an exception, you'll get a stack trace pointing to line 39. If `new BufferedReader(new FileReader(outFilename))` throws an exception, it's a little harder to tell if it was the `FileReader` creation or the `BufferedReader` creation.

Reply:
That's true.  Though, I think those constructors throw different exception types.